tensorboard/webapp/widgets/linked_time_fob/linked_time_fob_controller_component.ts

```diff
@@ -48,9 +36,8 @@ export class LinkedTimeFobControllerComponent {
   @ViewChild('startFobWrapper') readonly startFobWrapper!: ElementRef;
   @ViewChild('endFobWrapper') readonly endFobWrapper!: ElementRef;
   @Input() axisDirection!: AxisDirection;
-  @Input() steps!: number[];
   @Input() linkedTime!: LinkedTime;
-  @Input() temporalScale!: TemporalScale;
+  @Input() histogramFobData!: HistogramFobData;
```

Comment:
so would there another input for, say, `ScalarFobData`, or do you plan to use `FobCardData `
which is a combination for this two?

Reply:
I plan on combining them and replacing this with CardFobData.